src/strategies_and_params/replicastrategy.jl

```diff
@@ -61,46 +62,82 @@ c{i}_Op{k}_c{j} for operator overlaps.
 
 See [`lomc!`](@ref), [`ReplicaStrategy`](@ref) and [`AbstractHamiltonian`](@ref) (for an
 interface for implementing operators).
+
+If `transform` is not `nothing` then this strategy calculates the overlaps with respect to a 
+similarity transformation of the Hamiltonian (See e.g. [`GutzwillerSampling`](@ref).)
+```math
+    G = f H f^{-1}.
```

Comment:
In other parts of the documentation we have been putting hats on operators in LaTeX formulas, so I suggest we stay consistent here.
```suggestion
    \\hat{G} = f \\hat{H} f^{-1}.
```